### DISCUSSION TITLE
Unsure what these linting rules are asking for

### DISCUSSION THREAD

**Discussion by DennisTheMenace780** [eslint/eslint#16816] [Q&A]
So I just hopped on to a large project and got my `eslint_d` set up to read in from a `.eslintrc.js` file. Along my imports I keep getting hard-to-decipher messages that look like this:

```
eslint_d: Replace `useNavigation·}·from·"@react-navigation/nativ` with `styles·}·from·"./Today.styl`
eslint_d: Replace `format·}·from·"date-fns-tz` with `TodaysAppointment,·TodaysSchedule·}·from·"./components/TodaySchedule`
```
The errors are lighting up my imports and it's not obvious to me what this is asking from me. Would greatly appreciate any help if you've seen messages like this before. The nature of the messages makes it really challenging to craft a good Google search, so i'm tossing this up to the community.

**Comment by nzakas** [comment]
Those messages are coming from rules you have enabled. We don't maintain eslint_d, so I'm not sure why the messages are coming out so garbled. I'd suggest running the standard `eslint` CLI to get a cleaner view of the messages.
